### PR TITLE
New command resalloc-openstack-list

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by praiskup** [comment]
@frostyx wdyt?  First patch is for OSUOSL lab (we just need a list of running instances, volumes are not under our control), second one also for the PSI cloud (we occasionally remove just instance, not the assigned volumes - then after cmd_list detects also the removed instace, we need to cleanup the volumes).

File: resalloc_openstack/delete/main.py [hunk 1]
```diff
@@ -27,16 +27,17 @@
 from keystoneauth1.exceptions import ConnectionError
 
 def gather_tasks(gc):
+    server = None
     args = parser.parse_args()
 
     try:
         server = Server(nova, args.name)
+        gc.add('10_server', server)
```
Comment (FrostyX) [inline resalloc_openstack/delete/main.py:35]: There are magic values such as this one `10_server` and later down `05_volume`. Can you please move them out as constant and document their value?

**Comment by FrostyX** [comment]
Otherwise LGTM

File: resalloc_openstack/delete/main.py (hunk 1 above)
Comment (praiskup) [reply resalloc_openstack/delete/main.py:35]: These magic values are just to let the `gc` object remove them in the right (alphabetical order), no matter in which order the items are added.

Comment (FrostyX) [reply resalloc_openstack/delete/main.py:35]: Ah, that wasn't clear at first sight, and may be worth adding some comment to it. I thought they are values that you found somewhere in our OpenStack instance.

Comment (praiskup) [reply resalloc_openstack/delete/main.py:35]: Good point, I've documented the `GarbageCollector.add()` method, can you take one more look?

Comment (FrostyX) [reply resalloc_openstack/delete/main.py:35]: Perfect, it makes a lot of sense now.
+1

**Review by FrostyX — APPROVED** [review]

**Pull request merged**